### PR TITLE
feat(DefaultORMController)

### PR DESCRIPTION
**Pull request by tobiaskneubuehler**
one can now select the deleted field on PATCH (added to the getModel method of the ORMController)

please approve, so I can release

### CODE REVIEW

File: lib/DefaultORMController.js [hunk 1]
```diff
@@ -201,6 +201,11 @@
 
             query = (baseQuery || this.rootQuery)(['*'], idFilter);
 
+            // includeSoftDelted if deleted field is selected or filtered
+            if (queryData.getFields().indexOf('deleted') >= 0 || (queryData.filters.deleted && queryData.filters.deleted[0])) {
```
[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply lib/DefaultORMController.js]: https://github.com/eventEmitter/ee-soa-request/pull/4

[the remark this answers is not in the document]
Comment (tobiaskneubuehler) [reply lib/DefaultORMController.js]: Could contain dedicated criteria as objects or functions which are applied to the collection after retrieval.

[the remark this answers is not in the document]
Comment (eventEmitter) [reply lib/DefaultORMController.js]: :+1:

[the remark this answers is not in the document]
Comment (eventEmitter) [reply lib/DefaultORMController.js]: :+1:

**Pull request merged**